### ISSUE TITLE
Don't close issues that aren't fixed

### ISSUE THREAD

**Issue by r-tanner-f** [Armax/Pokemon-GO-node-api#151]
Please don't close issues that are not fixed. It's impossible to tell what's still broken, what's going to get fixed soon, and what needs some help. If something is in the works, maybe label it as such.

**Comment by tjhorner** [comment]
Agreed. That's not how the issue system is supposed to work. Of course, if it's someone asking for help then close it, since this isn't the right place to do so.
